docs/DeveloperGuide.md

```diff
@@ -451,6 +457,193 @@ Given below are instructions to test the app manually.
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
 
-1. Launch and Shutdown
-2. {Test case eg. Deleting a task}
-3. Saving Data (dealing with missing/corrupted data files)
+### 1. Launch and Shutdown
+1. Initial launch  
+i. Download the jar file and copy into an empty folder.
+ii. Double-click the jar file  
+
+### 2. Test Cases
+
+#### 2.1 Deleting a task
+2.1.1. Deleting a task while all tasks in the existing tasklist is listed.
+i. Prerequisites: User must be in the Timetable mode. Enter `mode /t` command to enter Timetable mode. List all tasks in the tasklist using the `list` command. There must be existing tasks in the list.  
+ii. Test case: `delete 1`    
+Expected: First task should be deleted from the tasklist.  
+iii. Test case: `delete 0`  
+Expected: No task is deleted. Error message will be printed in the command line interface.  
+iv. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the number of tasks in the tasklist)  
+Expected: No task is deleted. Error message will be printed in the command line interface. 
+
+#### 2.2 Adding a task
+
+2.2.1 Adding a task to the tasklist.  
+i. Prerequistes: User must be in the Timetable mode. Enter `mode /t` command to enter Timetable mode. 
+ii. Test case: `add /t test task /by 10-10-2020 2000`  
+Expected: Task is added into the tasklist with a success message printed.  
+iii. Test case: `add /t test task`  
+Expected: An error messsage will be printed in the command line interface.  
+iv. Test case: `add`  
+Expected: An error message along with a formatting guideline message will be printed.  
+v. Test case: `add /t test task /by 2020-10-10 2000`  
+Expected: An error message along with a formatting guideline message will be printed in the command line interface.
```

Comment:
The application will not return an error for having spaces. I'm not sure whether that was intended or missed restriction.

Reply:
it's intended. xD

Reply:
Oh even if the date formatted is wrong? I thought it needs to be in the format dd-MM-yyyy or can it be the other way around?

Reply:
oh wait as in the date format must be dd-MM-yyyy. It's just that both 1) `add /t test task /by 2020-10-10 2000`   and 2) `add /ttest task /by2020-10-10 2000`  will work! notice how there's a space between the delimiter and user input in 1).

Reply:
OHOH I SEE WHAT YOU MEAN NOW YOU'RE CORRECT.